packages/functoria-runtime/functoria-runtime.2.1.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer:   "Gabriel Radanne <drupyog@zoho.com>"
```

Comment:
@dbuenzli Well, it's more about reflecting the current situation, which is that functoria is collectively maintained by several mirage people. :)

Thanks @avsm !